src/pages/Assets/AssetDetails/AssetHistory.tsx

```diff
@@ -37,8 +37,8 @@ export const AssetHistory = ({ asset }: { asset: AssetMarketData }) => {
             </Center>
           }
         >
-          {data?.map((tx: Tx) => (
-            <TransactionRow key={`${tx.type}-${tx.txid}-${tx.asset}`} tx={tx} />
+          {data?.map((tx: Tx, id: number) => (
+            <TransactionRow key={id} tx={tx} />
```

Comment:
previous key was more robust

Reply:
This is causing an infinite loop

Reply:
I repeat, this is causing an infinite error message being thrown in the console

Reply:
Fixed the console issue by adding `key={0}` to the loader per their [docs](https://www.npmjs.com/package/react-infinite-scroller)